script/hassfest/icons.py

```diff
@@ -126,7 +126,9 @@ def validate_icon_file(config: Config, integration: Integration) -> None:  # noq
         integration.add_error("icons", f"Invalid JSON in {name}: {err}")
         return
 
-    schema = icon_schema(integration.integration_type)
+    overwrite_entity_component = name in ("notify", "image_processing")
```

Comment:
Not sure `overwrite_entity_component` is a good name.
I would place `("notify", "image_processing")` in a constant.

Reply:
Any suggestions?

Reply:
I see other parts of the code do not use a constant either